src/oaklib/utilities/lexical/lexical_indexer.py

```diff
@@ -310,24 +311,19 @@ def lexical_index_to_sssom(
         #            mappings.append(create_mapping(oi, term, r1, r2))
     logging.info("Done creating SSSOM mappings")
 
-    if meta is None:
-        meta = get_default_metadata()
-    if prefix_map:
-        meta.prefix_map.update(
-            {k: v for k, v in prefix_map.items() if k not in meta.prefix_map.keys()}
-        )
-    mapping_set_id = meta.metadata[MAPPING_SET_ID]
-    license = meta.metadata[LICENSE]
-
-    mset = MappingSet(mapping_set_id=mapping_set_id, mappings=mappings, license=license)
-    # doc = MappingSetDocument(prefix_map=oi.prefix_map(), mapping_set=mset)
-    doc = MappingSetDocument(prefix_map=meta.prefix_map, mapping_set=mset)
-    msdf = to_mapping_set_dataframe(doc)
-    num_mappings = len(msdf.df.index)
-    if ensure_strict_prefixes:
-        msdf.clean_prefix_map()
-        if len(msdf.df.index) < num_mappings:
-            raise ValueError("Mappings included prefixes that were not in the prefix map")
+    # TODO what's the business logic for combining these three
+    #  different streams of prefix maps?
+    converter = curies.chain(
```

Comment:
Need external guidance here - there are three streams of incoming prefix maps. How should they be prioritized?

Reply:
`oi.converter` can be last. the rest seems fine.

Reply:
Done in 17dfea14